index.html

```diff
@@ -1312,7 +1312,7 @@ <h3 id="Builder-wheres">Where Clauses</h3>
 </pre>
 
     <p id="Builder-forUpdate">
-      <b class="header">forUpdate</b><code>.transacting(t).forUpdate()</code>
+      <b class="header">forUpdate</b><code>.transacting(t).forUpdate([tables], [nowait])</code>
```

Comment:
Maybe this is a bit more clear spec of arguments `.forUpdate('table1', 'table2',... , [nowait])` parameter reading did seem so creative, that it could be good to explain with examples all the variations.